usama/sprint1/cdk.out/asset.0e9db55350e39158be9550d074e22f47df7cc37733a302a79954c0f2729738d0/web_health_publisher.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+import urllib3
+import urllib.request
+import json
+import constants
+
+
+def health_monitor(event, context):
+    
+    url = constants.URLS_TO_MONITOR
```

Comment:
- What if, we've there are a number of URLs in the constants file? 
- It's better to make the code generic